edit-post/components/header/plugins/index.js

```diff
@@ -0,0 +1,94 @@
+/**
+ * External dependencies
+ */
+import { map, isEmpty, isString } from 'lodash';
+import classnames from 'classnames';
+import { connect } from 'react-redux';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { withInstanceId, IconButton, MenuItemsGroup } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+import { getMoreMenuItems } from '../../../api/more-menu-item';
+import { getActivePlugin, getOpenedGeneralSidebar } from '../../../store/selectors';
+
+/**
+ * Renders a list of plugins that will activate different UI elements.
+ *
+ * @param   {Object} props The component props.
+ * @return {Object} The rendered list of menu items.
+ */
+function Plugins( props ) {
+	const ellipsisMenuItems = getMoreMenuItems();
```

Comment:
How does this component get updates when a new item is registered?

I'm starting to doubt that it should be its own standalone registry, and might benefit from using the [data module](https://github.com/WordPress/gutenberg/tree/master/data) (I've dabbled in considering this for blocks too, but it's not something I'm keen to take on 😉 ).

Reply:
Well, the API expects plugins to be registered before initialization, just like in the sidebar API, but it's worth considering moving this to the aforementioned store in an iteration. :)

Reply:
>Well, the API expects plugins to be registered before initialization, just like in the sidebar API, but it's worth considering moving this to the aforementioned store in an iteration. :)
>
>Also having the More Menu items change while the menu is open is not desirable from a UX perspective.

This just implies to me that we have no trust that the plugins header rendering is kept in sync with the overall state of the application (if we're considering registration of menu items to be part of said "state").